src/index.js

```diff
@@ -14,7 +19,11 @@ import injectJSXWrapper from "./helpers/runtime/jsx-wrapper";
 
 export default function ({ Plugin, types: t }) {
   return new Plugin("incremental-dom", { visitor : {
-    Program: setupInjector,
+    Program: function(program, parent, scope, file) {
+      const options = get(file, ['opts', 'extra', 'incremental-dom']);
+      setPrefix(options);
+      setupInjector(program, parent, scope, file);
```

Comment:
We were always setting up state in the `file` instance using `setupInjector` (used to track runtime-helper injections). We could pass around `file` everywhere, but I don't think it'd gain a whole lot.